compiler/treetab.nim

```diff
@@ -12,16 +12,18 @@
 import
   hashes, ast, astalgo, types
 
-proc hashTree(n: PNode): Hash =
-  if n == nil: return
+proc hashTree*(n: PNode): Hash =
+  if n.isNil:
+    return
   result = ord(n.kind)
   case n.kind
   of nkEmpty, nkNilLit, nkType:
     discard
   of nkIdent:
+    echo "what", "ident"
```

Comment:
Some debugging code left here.

Reply:
That should've been an assert: I wondered it it's possible to have a nkIdent there at all at this place in sempass